src/dlgTriggerEditor.cpp

```diff
@@ -3469,18 +3469,22 @@ void dlgTriggerEditor::addVar(bool isFolder)
         mpVarsMainArea->comboBox_variable_value_type->setCurrentIndex(0);
     }
 
-    QStringList nameL;
-    nameL << QString();
-    QTreeWidgetItem* cItem = treeWidget_variables->currentItem();
     LuaInterface* lI = mpHost->getLuaInterface();
     VarUnit* vu = lI->getVarUnit();
-    TVar* cVar = vu->getWVar(cItem);
+
+    QStringList nameL;
+    nameL << QString(isFolder ? "New table name" : "New variable name");
```

Comment:
Given that this is user-facing I think you will need to mark these texts as translatable... (put in `tr(`...`)` wrappers)!

Reply:
good point :)